llm/models/transformer.py

```diff
@@ -217,6 +217,7 @@ def step(self) -> None:
         assert self.optimizer, "Cannot take an optimization step with optimizer=None"
 
         self.unembedding_layer.step()
+        self.final_norm.step()
```

Comment:
was missing before. training still works because in LayerNorm, gamma is initialized to 1 and beta to 0, so it's not a big deal if it's not trained.